kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableReadsSuite.scala

```diff
@@ -38,6 +39,21 @@ import scala.collection.JavaConverters._
 
 class DeltaTableReadsSuite extends AnyFunSuite with TestUtils {
 
+  test("CRC metadata & protocol fetching") {
+    withTempDir { dir =>
+      val path = dir.getCanonicalPath
+      withSQLConf(DeltaSQLConf.DELTA_WRITE_CHECKSUM_ENABLED.key -> "true") {
+        spark.sql(s"CREATE TABLE delta.`$path` USING DELTA AS " +
+        s"SELECT 0 as value")
+      }
+
+      val snapshot = Table.forPath(defaultEngine, path)
+        .getSnapshotAsOfVersion(defaultEngine, 0)
+
+      readSnapshot(snapshot)
```

Comment:
this doesn't really prove that we got the value from the CRC file. if the CRC file was not written, we would still have been able to load version 0. are there any metrics we could collect to help indicate this? perhaps sync with @allisonport-db

Reply:
I honestly don't know if we need this test -- I think this is tested in `LogReplayEngineMetricsSuite`

Reply:
A possible follow-up task is to add crc-specific info/metrics to `SnapshotReport`

Reply:
Sounds good. Let's remove this test for now. @huan233usc can you please track this as an item in your project plan to add metrics for CRC reading so we can properly test this aspect later?

Reply:
SG and revert this test for now and will add metric and tests back seperately